.github/workflows/build_wheels.yml

```
@@ -88,7 +88,7 @@ jobs:
       - run: bash ./ci/triage_build.sh "${{ matrix.arch }}" "${{ github.event.pull_request.head.sha || github.sha }}" "${{ matrix.python }}"
 
       # Now actually build the wheels
-      - uses: pypa/cibuildwheel@v3.1.1
+      - uses: pypa/cibuildwheel@v3.1.2
```

```suggestion
      - uses: pypa/cibuildwheel@9e4e50bd76b3190f55304387e333f6234823ea9b  # 3.1.2
```

Can we move to pinning to SHAs?

---

https://github.com/h5py/h5py/pull/2624